app/index.tsx

```diff
@@ -2,8 +2,21 @@ import React, { Fragment } from 'react';
 import { ThemeProvider, theme } from '@blockstack/ui';
 import { render } from 'react-dom';
 import { AppContainer as ReactHotAppContainer } from 'react-hot-loader';
+import { DefaultOptions, QueryClient, QueryClientProvider } from 'react-query';
 
 import { configureStore, history } from './store/configureStore';
+import { DEFAULT_POLLING_INTERVAL } from './constants';
+
+const config: DefaultOptions['queries'] = {
+  refetchInterval: DEFAULT_POLLING_INTERVAL,
+  keepPreviousData: true,
+  refetchOnWindowFocus: true,
+  staleTime: 30_000,
```

Comment:
`refetchInterval: 60_000,` but `staleTime: 30_000,`? I think the stale time should be at least equal to the interval, what do you think?

Reply:
My rationale for setting this prop and the one above was to stop over-spamming expensive requests. Jumping to and fro the window always trigger a `delegation-status` req.

I don't object to increasing stale time in this case, though not sure that's a hard rule to follow. Perhaps you want really slow background polling so idle clients don't hammer the server, but want often updates during use? 🤷🏼

Reply:
Ah I was mistaken, I assumed DEFAULT_POLLING_INTERVAL was 60 seconds, and thought then the stale time would be half. disregard my comment